eventlet/green/http/client.py

```diff
@@ -1126,7 +1126,7 @@ def _send_output(self, message_body=None, encode_chunked=False):
 
                 if encode_chunked and self._http_vsn == 11:
                     # chunked encoding
-                    chunk = '{0:X}\r\n'.format(len(chunk)).encode('ascii') + chunk + b'\r\n'
+                    chunk = '{:X}\r\n'.format(len(chunk)).encode('ascii') + chunk + b'\r\n'
```

Comment:
The current version LGTM, but, since we support versions of python higher than 3.7,  f-string could be used for this kind code.